README.md

```diff
@@ -15,11 +15,12 @@ Before following the steps be sure to clone this repository to the local machine
 
 1. Download & Install [Docker](https://www.docker.com/products/docker-desktop).
 2. Download & Install [PostgreSQL](https://www.postgresql.org/download/)
-3. Build the Docker images:
+3. Login to docker: Try `docker login` first, or if using bash `winpty docker login`. Note it asks for **username**, not email.
+4. Build the Docker images:
     1. Navigate to `/BackEnd/` in your terminal.
     2. Run `docker build -f docker/admin.core.Dockerfile -t unosquare/admincore:latest . --no-cache`
     3. Run `docker build -f docker/express.cors.proxy.Dockerfile -t unosquare/cors-proxy:latest . --no-cache`
```

Comment:
I know this isn't part of the updates, but it'd be nice if we set up the docker-compose build